src/20_Components/amp-experiment.html

```diff
@@ -0,0 +1,107 @@
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+   <!-- #### Setup -->
+  <!--
+    Import the `amp-experiment` component.
+  -->
+  <script async custom-element="amp-experiment" src="https://cdn.ampproject.org/v0/amp-experiment-0.1.js"></script>
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+  <!--
+    Style different variants of the experiment by using `body[amp-x-name-of-experiment="nameOfVariant"`].
+  -->
+  <style amp-custom>
+    body[amp-x-button-color-experiment="yellow"] .test-button{
+      background-color: yellow;
+      color: black;
+    }
+    body[amp-x-button-color-experiment="red"] .test-button{
+      background-color: red;
+    }
+    body[amp-x-button-color-experiment="blue"] .test-button{
+      background-color: blue;
+    }
+    button {
+      background-color: grey;
+    }
+  </style>
+</head>
+<body>
+
+  <!-- #### Basic Usage -->
+  <!--
+    Configure experiments inside the `amp-experiment` inside a JSON object. Use `variants` within each experiment to declare how many users in percent will be part of an experiment variant. The total sum of the variants must be a number <= 100, if the number is < 100, then the default behaviour is used. A user belongs to one of the variants based on a random number generated between 0 and 100. For more details about variants read the official [documentation](https://github.com/ampproject/amphtml/blob/master/extensions/amp-experiment/amp-experiment.md). Multiple experiments can be run on the same AMP document in parallel with their own sets of variants. One AMP document can have at most one `amp-experiment` element.
+  -->
+  <amp-experiment>
+    <script type="application/json">
+      {
+        "button-color-experiment": {
+          "variants": {
+            "yellow": 30,
+            "red": 30,
+            "blue": 30
+          }
+        }
+      }
+    </script>
+  </amp-experiment>
+</amp-experiment>
+<!--
+  An experimental button. The background of the button is going to be yellow for the 30% of users, red for 30% of users, blue for 30% of users and, for the remaining 10% of users, the button background is going to be the default color which is grey in this sample.
+-->
+<button class= "button test-button">Click here</button>
+
+<!-- #### Reporting -->
+
+<!--
+  Measure results of the experiment using [amp-pixel](https://github.com/ampproject/amphtml/blob/master/builtins/amp-pixel.md) and/or [amp-analytics](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/amp-analytics.md). Allocated variants are available as a URL substitution variable: `VARIANT(name-of-the-experiment)`. You can check the analytics requests in the network tab of the developer tools.
```

Comment:
Suggestion: split the paragraph into two sections aligned with the two analytics tags:

```
<!--
 +  Measure results of the experiment using [amp-pixel](https://github.com/ampproject/amphtml/blob/master/builtins/amp-pixel.md) ...
-->
<amp-pixel src="http://example.com/track/?xname=button-color-experiment&xvar=VARIANT(button-color-experiment)"></amp-pixel>
<!--
... and/or [amp-analytics](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/amp-analytics.md). Allocated variants are available as a URL substitution variable: `VARIANT(name-of-the-experiment)`. You can check the analytics requests in the network tab of the developer tools.
-->
<amp-analytics type="googleanalytics" id="analytics">
...
```